tests/internal/test_deps.py

```diff
@@ -79,6 +80,13 @@ def test_gstreamer_info(self):
         assert gi.__version__ in result["other"]
         assert "Relevant elements:" in result["other"]
 
+        with mock.patch(
```

Comment:
This would probably belong in its own test with a self documenting name but I'm not sure this actually adds useful testing of our functionality.

Reply:
It is needed for the 100% coverage.

Reply:
Yeh, I figured that might be the reason. An improvement could be to check it was in the right section, i.e. before "Not found:" list entry. I appreciate that's a bit fiddly, never mind.

Reply:
I aim to start removing the "if TYPECHECKING" later this month (once the outstanding PR is merged) and add 100%. coverage to the audio directory...so any change is short lived.